src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @Min (value = 20)
     private int capacity;
+    @NotNull
+    @Size(min = 20, max = 150)
     private String description;
```

Comment:
description in fact can be an optional field, but every hall has a capacity

Reply:
amended